Flatiron - Tuples.playground/Contents.swift

```diff
@@ -0,0 +1,67 @@
+//: Playground - noun: a place where people can play
+
+import UIKit
+
+var str = "Hello, playground"
+
+let sampleTuple = ( alpha:34, beta:"who?", gamma:[1, 4, 16 ] )
+
```

Comment:
Here is the [style guide](https://github.com/raywenderlich/swift-style-guide) I want to reference throughout this curriculum.

I would prefer to do the spacing as follows (which is what I'm stressing to all the contractors), so that way we're all on the same page.

```swift
let sampleTuple = (alpha: 34, beta: "who?", gamma: [1, 4, 16 ])
```